README.md

```diff
@@ -31,40 +31,48 @@ used in combination with an Ethereum client via the `eth.compile.solidity()` RPC
 
 ### Usage in Projects
 
-#### From early versions
-
-It can also be included and used in other projects:
-
-```javascript
-var solc = require('solc')
-var input = 'contract x { function g() {} }'
-// Setting 1 as second paramater activates the optimiser
-var output = solc.compile(input, 1)
-for (var contractName in output.contracts) {
-	// code and ABI that are needed by web3
-	console.log(contractName + ': ' + output.contracts[contractName].bytecode)
-	console.log(contractName + '; ' + JSON.parse(output.contracts[contractName].interface))
-}
-```
+#### From version 0.5.0
 
-#### From version 0.1.6
+Starting from version 0.5.0, `compile`, `compileStandard` and `compileStandardWrapper` all do the same thing - what `compileStandardWrapper` used to do.
```

Comment:
Perhaps we should briefly say what they do and not assume people knew the old versions.